moveit_core/kinematics_base/src/kinematics_base.cpp

```diff
@@ -92,6 +92,23 @@ bool KinematicsBase::initialize(const rclcpp::Node::SharedPtr& /*node*/,
   return false;
 }
 
+bool KinematicsBase::initialize(const node_interface::NodeInterfaceSharedPtr& node_interface, const moveit::core::RobotModel& robot_model, const std::string& group_name,
+                                const std::string& base_frame, const std::vector<std::string>& tip_frames,
+                                double search_discretization)
+{
+    if (auto node = node_interface->get_rcl_node()){
```

Comment:
Is this correct? From my understanding an error is now returned if `node != nullptr`. Shouldn't this be something like
```
auto node = node_interface->get_rcl_node();
if(node == nullptr){
...
return false;
} else { ***Do something with node?***}
```

Reply:
Good catch! The if/else was flipped.